cloudformation/policies/parallelcluster-policies.yaml

```diff
@@ -179,6 +183,8 @@ Resources:
   ParallelClusterLambdaRole:
     Type: AWS::IAM::Role
     Properties:
+      PermissionsBoundary: !If [EnablePermissionsBoundary, !Ref PermissionsBoundaryPolicy, !Ref 'AWS::NoValue']
```

Comment:
*Future improvement*
If you use `AWS::NoValue` as default value for the permission boundary parameter, then you will be able to avoid the If statement around the condition `EnablePermissionsBoundary` because you would simply assign the `PermissionsBoundaryPolicy` directly.

We can consider this as out of scope for this PR because you're proposing a solution that is consistent with the rest of the implementation. We should evaluate the above improvement on the whole template.